app/build.gradle

```diff
@@ -84,7 +84,7 @@ dependencies {
     implementation 'com.jakewharton.timber:timber:5.0.1'
 
     // Jetpack components
-    def lifecycle_version = "2.5.0-beta01"
+    def lifecycle_version = "2.5.0-rc01"
```

Comment:
Több helyen is vannak verzióváltozások, ezek kapcsolódnak ehhez az issue-hoz?

Reply:
Közvetlenül nem, csak dependency upgradek, viszont pontosan nem tudom, hogyan kell ezeket az upgradeket kezelni. Main ágba illik pusholni őket?

Reply:
Hát én eleve nem frissítenék soha snapshot verziót, mert azok nem stabilak és rengeteg overhead-del jár. De amúgy külön pr-be érdemes szerintem.